aws/instance-scheduler/aws-instance-scheduler.tf

```diff
@@ -44,12 +44,12 @@ resource "aws_cloudformation_stack" "schedule" {
 module "instance_scheduler_agent" {
   source = "./instance-scheduler-agent"
 
-  create            = "${local.creat_instance_scheduler_agent}"
+  create            = "${local.create_instance_scheduler_agent}"
   master_account_id = "${var.master_account_id}"
 }
 
 locals {
-  creat_instance_scheduler                   = "${var.create == 1 ? local.is_master : 0 }"
-  creat_instance_scheduler_agent             = "${var.create == 1 ? local.is_child : 0 }"
-  aws_instance_scheduler_cross_account_roles = "${join(",",(formatlist("arn:aws:iam::%s:role/aws-instance-scheduler-re-EC2SchedulerCrossAccount", var.accounts)))}"
+  create_instance_scheduler                  = "${coalesce(var.is_standalone_scheduler, local.is_master)}"
```

Comment:
These types of create resources should include logic from the `create` variable.